package.json

```diff
@@ -48,6 +48,7 @@
     "jest": "^25.5.4",
     "jest-cli": "^25.5.4",
     "parcel-bundler": "^1.12.4",
+    "path-browserify": "^1.0.1",
```

Comment:
This should be a regular dependency, right? Not a dev dependency?

Reply:
yep, good shout!